src/resolvers-cassandra/Settings/mutations.js

```diff
@@ -468,6 +518,7 @@ module.exports = {
   createSite: trackEvent(createSite, 'createSite'),
   removeSite: trackEvent(removeSite, 'removeSite'),
   createStream: trackEvent(createStream, 'createStream'),
+  editSite: trackEvent(withRunTime(editSite), 'editSite'),
```

Comment:
Is it worth having a separate editSite when it can be implemented client-side by first issuing rmeoveSite and then createSite?